test/jdk/java/awt/TrayIcon/TrayIconScalingTest.java

```diff
@@ -85,28 +85,27 @@ public static void main(String[] args)
             System.out.println("SystemTray is not supported");
             return;
         }
-        PassFailJFrame passFailJFrame = PassFailJFrame.builder()
-                .title("TrayIcon Test Instructions")
-                .instructions(INSTRUCTIONS)
-                .testTimeOut(8)
-                .rows(25)
-                .columns(70)
-                .screenCapture()
-                .build();
 
-        createAndShowGUI();
-        // does not have a test window,
-        // hence only the instruction frame is positioned
-        PassFailJFrame.positionTestWindow(null,
-                PassFailJFrame.Position.HORIZONTAL);
+        createAndShowTrayIcon();
+
         try {
-            passFailJFrame.awaitAndCheck();
+            PassFailJFrame.builder()
+                    .title("TrayIcon Test Instructions")
+                    .instructions(INSTRUCTIONS)
+                    .testTimeOut(8)
+                    .rows(25)
+                    .columns(70)
```

Comment:
Personally I like rows+columns to immediately follow instructions,
but I'll approve anyway and let you decide.

Reply:
I haven't changed these lines, only indentation is changed because I moved it into `try` block.

I prefer keeping it as is so that these lines are attributed to Lawrence who modified the lines when he introduced the builder.